examples/twitter.amp.html

```diff
@@ -20,30 +20,45 @@
       layout="responsive"
       data-tweetid="638793490521001985">
       <blockquote placeholder class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I only needed to change some CSS. <a href="http://t.co/LvjLbjgY9F">pic.twitter.com/LvjLbjgY9F</a></p>&mdash; Malte Ubl (@cramforce) <a href="https://twitter.com/cramforce/status/638793490521001985">September 1, 2015</a></blockquote>
+      <div fallback>
+        Something happened..
```

Comment:
What I understood from the discussion in #9355 is that the requested feature is a fallback that only appears when there's a problem (while the current behavior using a `placeholder` makes the error message appear while the element is loading and persist if the error is detected or disappear if the error wasn't detected). Let's discuss this offline whenever you have time.